src/apps/weblib/typeschema-api/styles.yaml

```diff
@@ -1,14 +1,21 @@
 ---
 definitions:
-    Length:
+    LengthUnit:
```

Comment:
Erre miért van szükség? 
Általánosságban: a style-oknál a vezérelv az volt, hogy amennyire csak lehet, próbáljunk meg CSS kompatibilisek maradni. Ez, hogy bevezetjük a + operátort a length tipusú propertiknél túl mutat ezen.
Ha nem must have, én nem csinálnám. Ha must have, akkor CSS kompatibilisen a calc()-ról kellene mintázni: https://developer.mozilla.org/en-US/docs/Web/CSS/calc

Reply:
Én próbáltam a toString/fromString-jét a Gfx::Length-nek egységesíteni, mert a toString (operator std::string) így működik ('+' jellel elválaszva a 3 komponenst (%/px/em) kiírja).

ennek nem feltétlenül kell kifelé is látszania

Reply:
Ah értem.
Akkor az bizti, hogy kifelé nem kellene látszania.
Kicsit kilóg nekem ebből az osztályból, hogy kifejezést értékel ki.